--- __init__.py	2018-10-02 20:52:24.000000000 +0000
+++ __init__.changed.py	2019-03-12 12:44:36.877849896 +0000
@@ -53,6 +53,7 @@
     '#cloud-config-archive': 'text/cloud-config-archive',
     '#cloud-config-jsonp': 'text/cloud-config-jsonp',
     '## template: jinja': 'text/jinja2',
+    '#taupage-ami-config': 'text/x-taupage-ami-config',
 }
 
 # Sorted longest first
@@ -197,9 +198,10 @@
         LOG.debug('content_type "%s" is excluded', content_type)
         return
 
-    if content_type in PART_CONTENT_TYPES:
-        walker_handle_handler(data, content_type, filename, payload)
-        return
+    # we don't want custom part handlers at all
+    #if content_type in PART_CONTENT_TYPES:
+    #    walker_handle_handler(data, content_type, filename, payload)
+    #    return
     handlers = data['handlers']
     if content_type in handlers:
         run_part(handlers[content_type], data['data'], filename,

