--- stages.py	2018-10-02 20:52:24.000000000 +0000
+++ stages.changed.py	2019-03-12 10:53:51.212415699 +0000
@@ -22,6 +22,7 @@
 from cloudinit.handlers.jinja_template import JinjaTemplatePartHandler
 from cloudinit.handlers.shell_script import ShellScriptPartHandler
 from cloudinit.handlers.upstart_job import UpstartJobPartHandler
+from cloudinit.handlers.zalando_ami_config import ZalandoAMIConfigPartHandler
 
 from cloudinit.event import EventType
 
@@ -417,10 +418,12 @@
         cloudconfig_handler = CloudConfigPartHandler(**opts)
         shellscript_handler = ShellScriptPartHandler(**opts)
         def_handlers = [
-            cloudconfig_handler,
-            shellscript_handler,
-            BootHookPartHandler(**opts),
-            UpstartJobPartHandler(**opts),
+            # yes, it sucks that this is static
+            #cloudconfig_handler,
+            #shellscript_handler,
+            #BootHookPartHandler(**opts),
+            #UpstartJobPartHandler(**opts),
+            ZalandoAMIConfigPartHandler(**opts),
         ]
         opts.update(
             {'sub_handlers': [cloudconfig_handler, shellscript_handler]})
